billing.md

```diff
@@ -99,34 +93,119 @@ Next, add the `Billable` trait to your model definition. This trait provides var
         use Billable;
     }
 
+Cashier assumes your Billable model will be the default `App\User` class that ships with Laravel. If you wish to change this you can update this in your `.env` file:
+
+    CASHIER_MODEL=App\User
```

Comment:
Hi Dries, i think it's worth documenting that changing this will also change the foreign key lookup column on the Subscriptions table - for example, i changed this to `App\Team` for my app, and i got an error because it was looking to match the `team_id` column on subscriptions instead of `user_id`

Reply:
Also worth noting that in the other migration, you should change `users` table name to whichever entity you're using.